prow/plugins/label/label_test.go

```diff
@@ -20,10 +20,11 @@ import (
 	"fmt"
 	"testing"
 
+	"reflect"
```

Comment:
In general it's bad form to use the reflect package at all, including in tests. Sometimes I allow it just because it's easier, but it's a bad sign.